### PR TITLE
For #885;

### PR DESCRIPTION
**Pull request by HenkKodde**
- updated local/hosts config to include newly released services
- updated dependancies on package.json
- removed caret's in package-lock.json.

### CODE REVIEW

**Comment by HenkKodde** [comment]
Have tested this on my local environment as per the guidelines in .gitbook.md. Everything works fine and able to view the updated content on my local browser without and issues.

info: >> generation finished with success in 148.1s ! 

Starting server ...
Serving book on http://localhost:8989

Will need some assistance in resolving the circleCI error.

**Comment by HenkKodde** [comment]
All checks passed - story isa ready for review.

File: deployment-guide/README.md [hunk 1]
```diff
@@ -209,7 +209,7 @@ Please review [Mojaloop Helm Chart](../repositories/helm.md) to understand the r
    
    Include the following line to the config;
    ```text
-   127.0.0.1       interop-switch.local central-kms.local forensic-logging-sidecar.local central-ledger.local central-end-user-registry.local central-directory.local central-hub.local central-settlements.local ml-api-adapter.local
+   127.0.0.1       interop-switch.local central-end-user-registry.local central-directory.local central-hub.local central-ledger forensic-logging-sidecar central-kms.local central-settlement.local central-event-processor.local ml-api-adapter.local email-notifier.local account-lookup-service.local quoting-service.local moja-simulator.local
```
Comment (elnyry) [inline deployment-guide/README.md]: From Helm version 6.2.x I think, we do not have interop-switch central-end-user-registry and central-directory anymore.. 

Can this be modified to reflect that information, i.e., that if Helm versions >= 6.2.2 are being used then the above mentioned services are not needed (and that they're only needed for helm versions before 6.2.2)?

Comment (HenkKodde) [reply deployment-guide/README.md]: Thank you for the review - Will action accordingly and include the appropriate note.

Comment (HenkKodde) [reply deployment-guide/README.md]: Believe central-hub can also be removed.

**Pull request merged**